Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node
```

maybe we should use the slim version unless we are using unique modules.
```suggestion
FROM node
```
FROM node:18.11-slim